src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationScopeParsingTests.cs

```diff
@@ -1162,29 +1162,41 @@ public void Lambda_06()
         {
             string source = "(ref scoped scoped R r) => { }";
             UsingExpression(source, TestOptions.Regular11,
-                // (1,1): error CS1073: Unexpected token 'scoped'
-                // (ref scoped scoped R r) => { }
-                Diagnostic(ErrorCode.ERR_UnexpectedToken, "(ref scoped ").WithArguments("scoped").WithLocation(1, 1),
-                // (1,2): error CS1525: Invalid expression term 'ref'
-                // (ref scoped scoped R r) => { }
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "ref scoped").WithArguments("ref").WithLocation(1, 2),
-                // (1,13): error CS1026: ) expected
+                // (1,20): error CS1003: Syntax error, ',' expected
                 // (ref scoped scoped R r) => { }
-                Diagnostic(ErrorCode.ERR_CloseParenExpected, "scoped").WithLocation(1, 13)
-                );
+                Diagnostic(ErrorCode.ERR_SyntaxError, "R").WithArguments(",").WithLocation(1, 20));
```

Comment:
better parsing. t his is now parsed as `(ref scoped scoped, R r) => { }` (a missing comma in a lambda).